src/site/_utils/_hooks/authors.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const addPagination = require('../add-pagination');
+
+/**
+ * @param {Author[]} authors
+ * @return {Author[]}
+ */
+const authorsFeed = (authors) => {
+  let filteredAuthors = [];
+
+  if (process.env.ELEVENTY_ENV !== 'dev') {
```

Comment:
Whenever possible I think doing an early return can be better than doing a not condition. It's a bit less cognitive overhead.  So another way to write this would be:

```js
let filteredAuthors = [];
if (process.env.ELEVENTY_ENV === dev) {
  return filteredAuthors;
}

filteredAuthors = authors.reduce(...);
return filteredAuthors.
```